### PR TITLE
Add nicknaming and releasing

### PR DESCRIPTION
**Pull request by Wolvan**
Allow Pokemon to be nicknamed or even released if this is what a user
wants. For this to work properly, storage had to be changed slightly
This also leads to being able to capture multiple Pokemon of a single
species and gender.

Closes 2 points of #855

### CODE REVIEW

File: Extensions/pokes.css [hunk 1]
```diff
@@ -115,7 +115,6 @@
 	content: "";
```
Comment (nightpool) [inline Extensions/pokes.css]: (here in the most recent pull)

Comment (Wolvan) [inline Extensions/pokes.css]: As I said, I did not know what the unicode is, I copied it from tumblr's stylesheet

Comment (ThePsionic) [inline Extensions/pokes.css]: a

Comment (Wolvan) [inline Extensions/pokes.css]: Thanks Psi, you're a bro. Fixed

**Comment by ThePsionic** [comment]
LGTM! Good job @Wolvan 
@homu r+

**Pull request merged**